tasties_app/migrations/0001_initial.py

```diff
@@ -0,0 +1,83 @@
+            ],
+        ),
+        migrations.CreateModel(
+            name='Recipe',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=64, unique=True,
+                                           validators=[django.core.validators.MinLengthValidator(1)])),
+                ('description', models.CharField(max_length=512,
+                                                 validators=[django.core.validators.MinLengthValidator(1)])),
+                ('directions', models.CharField(max_length=65536,
+                                                validators=[django.core.validators.MinLengthValidator(1)])),
+                ('publication_date', models.DateField(default=django.utils.timezone.now)),
+                ('minutes_to_make', models.IntegerField(validators=[django.core.validators.MinValueValidator(1)])),
+                ('recipe_picture', models.ImageField(upload_to='recipe_pictures')),
+                ('author_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE,
+                                                to=settings.AUTH_USER_MODEL)),
+                ('categories', models.ManyToManyField(to='tasties_app.category')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Rating',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('rating', models.IntegerField(validators=[django.core.validators.MinValueValidator(1),
+                                                           django.core.validators.MaxValueValidator(5)])),
+                ('author_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE,
+                                                to=settings.AUTH_USER_MODEL)),
+                ('recipe_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='tasties_app.recipe')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Ingredient',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('amount', models.DecimalField(decimal_places=2, max_digits=10,
+                                               validators=[django.core.validators.MinValueValidator(0.0001)])),
+                ('measurement_unit', models.CharField(choices=[('WHOLE', 'Whole'), ('FLOZ', 'Fluid Ounce'),
+                                                               ('TSP', 'Teaspoon'), ('OZ', 'Ounce'), ('CUP', 'Cup'),
+                                                               ('GRAM', 'Gram'), ('ML', 'Milliliter'),
+                                                               ('KG', 'Kilogram'), ('L', 'Liter'), ('U', 'Units')],
```

Comment:
It looks like you've updated the models.py file to include more measurement units choices for an ingredient. Since you've updated models.py, the other team members working off of my model stubs PR don't have these changes in their models.py - so you have to include your updated models.py if you're going to upload an initial migration referencing it. We may run into serious problems if we don't sync everyone's models.py to this updated version.

Reply:
Thanks. Fixed

Reply:
I see you've removed the changes that you've added (at least it doesn't seem like the migration is referencing them). Very well, I think this should work. We shouldn't worry about a lack of measurement unit choices - more measurement units (such as Liter, Kilogram, Pound, etc. ) can be derived (in logic) from what we have.